test/contrib/spark_test.py

```diff
@@ -193,7 +193,7 @@ def test_app_interruption(self, proc):
         def interrupt():
             raise KeyboardInterrupt()
 
-        proc.return_value.poll = interrupt
+        proc.return_value.wait = interrupt
```

Comment:
btw ideally we should use the mock library for this. but that's prob outside the scope of this PR